packages/backend/src/machine-utils.ts

```diff
@@ -21,15 +21,50 @@ import * as os from 'node:os';
 import * as path from 'node:path';
 import * as fs from 'node:fs';
 import { satisfies, coerce } from 'semver';
+import type { ContainerProviderConnection } from '@podman-desktop/api';
+
+function getMachineProviderEnv(connection: extensionApi.ContainerProviderConnection): string {
+  switch (connection.vmType) {
```

Comment:
maybe we should lowercase vmType before checking so we don't have to add `Wsl` as well?

but just a nitpick, otherwise LGTM.

Reply:
I let it on purpose because the 2 values (wsl and Wsl) are there because of a bug in Podman Desktop core that replace the lowercase vmType by the uppercase one in order to get a better displayed value but that's incorrect and in a future only wsl (as well as libkrun) will be sent in 1.13

Reply:
no problem! lgtm